Resources/Prototypes/_ES/Entities/Markers/Spawners/Distributed/cargo.yml

```
@@ -0,0 +1,50 @@
+- type: entity
+  id: ESSpawnerRandomDistributedCargo
+  parent: ESMarkerBase
+  name: Cargo Crate Spawner
+  components:
+  - type: Sprite
+    layers:
+    - state: crate
+    - state: random
```

```suggestion
    - state: rand-dist
```

as a distributed spawner, should use the `rand-dist` state instead of `random`